storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/put/DefaultPutResolver.java

```diff
@@ -65,7 +65,7 @@ private PutResult updateOrInsert(@NonNull StorIOSQLiteDb storIOSQLiteDb, @NonNul
         final int numberOfUpdatedRows = storIOSQLiteDb.internal().update(
                 new UpdateQuery.Builder()
                         .table(table)
-                        .where(BaseColumns._ID + "=?")
+                        .where(getIdFieldName() + "=?")
```

Comment:
What about to pass `idFieldName` to `updateOrInsert()` as parameter for better performance?